content/compose/profiles.md

```diff
@@ -71,6 +69,10 @@ The above commands would both start your application with the `debug` profile en
 In the example, `compose.yml` file above, this starts the services `backend`,
 `db` and `phpmyadmin`.
 
+You can also enable
+multiple profiles, e.g. with `docker compose --profile frontend --profile debug up`
+the profiles `frontend` and `debug` will be enabled.
+
```

Comment:
Should we also talk about `--profile "*"` which activates all the profiles, WDYT?